spec/helpstation/processors/by_key_fetcher_spec.rb

```diff
@@ -22,7 +22,7 @@
       expect(subject).to be_a(Substation::Response::Success)
       expect(subject.output).to eq(
         my_input: input[:my_input],
-        my_output: output
+        my_output: 'test and {:mock_service=>:mock_value}'
```

Comment:
I agree, it should be local then. Actually I don't see a reason why `env` at the moment is global at all.